fixtures/uat-scenarios/README.md

```diff
@@ -0,0 +1,113 @@
+| EXTERNAL_CORRELATION_ID | A random UUID                          |
+| COURT_LOCATION          | A random court code                    |
+| DATE_OF_HEARING         | The current date                       |
+| PROSECUTOR_REFERENCE    | An incrementally generated ASN         |
+| PNC_IDENTIFIER          | "20230012345P"                         |
+| _PTIURN_                | "01XX" + random 7 digit number         |
+| GIVEN_NAME              | A random first name                    |
+| FAMILY_NAME             | A random surname                       |
+| DATE_OF_BIRTH           | "1983-03-11"                           |
+| ADDRESS_LINE_1          | A random street and house number       |
+| ADDRESS_LINE_2          | A random street name                   |
+| ADDRESS_LINE_3          | A random city                          |
+| ADDRESS_LINE_4          | A random county                        |
+| ADDRESS_LINE_5          | A random postcode                      |
+| OFFENCE_LOCATION        | A random address, cut to 80 characters |
+
+### Anonymising Incoming Message
+
+- Replace the value of `<CorrelationID>` with `EXTERNAL_CORRELATION_ID`
+
+The message content is contained withing the `<ns2:DataStreamContent>` tag. To make editing easier, copy the value of this tag to a new file and run it through an xml formatter.
+
+Older cases may have escaped the xml by replacing `<` with `&lt;` and `>` with `&gt;`. You can use a find/replace to change the message content back to xml.
+
+Replace the following fields:
+
+- Replace the value of `<CourtHearingLocation>` with `COURT_LOCATION`
+- Replace the value of `<DateOfHearing>` with `DATE_OF_HEARING`
+- Replace the value of `<PTIURN>` with `_PTIURN_`
+- Replace the value of `<ProsecutorReference>` with `PROSECUTOR_REFERENCE`
+- Replace the value of `<PersonGivenName1>` with `GIVEN_NAME` - Replace any additional given names with the same string
+- Replace the value of `<PersonFamilyName>` with `FAMILY_NAME`
+- Replace the value of `<Birthdate>` with `DATE_OF_BIRTH`
+- Replace the value of `<PNCidentifier>` with `PNC_IDENTIFIER`
+- Replace the values of `<AddressLine1>` through `<AddressLine5>` with the appropriate `ADDRESS_LINE_*` string
+- Check any `<OffenceWording>` for names or addresses, replace with appropriate strings for the seed script to replace
+- Replace any `<ConvictionDate>` tags with `DATE_OF_HEARING`
+
+Once finished, It is a good idea to have another team member check over the xml in case anything has been missed. Do this via screen share, do not send over slack or email.
+
+Once you are confident that no `PII` remains, minify the xml and paste back into the `incoming-message` file.
+
+### Anonymising PNC Response
+
+- Replace the value of `<FSC>` with `K01XM`
+- Replace the value of `<IDS>` with `K00/000000X FAMILY_NAME`
+
+Once these two files have beend anonymised, you can use your local dev environment to check that they produce the type of exceptions/triggers that you expect.
```

Comment:
Might be worth a note here saying that these fields are expected to be a specific size so if you're not getting the expected result, check for errors in Bichard to do with this